drltrace/scripts/gen_drltrace_config.py

```diff
@@ -0,0 +1,537 @@
+"INPUT_RECORD":"INPUT_RECORD",
+"HEAP_INFORMATION_CLASS":"HEAP_INFORMATION_CLASS",
+"GEOTYPE":"GEOTYPE",
+"GEOCLASS":"GEOCLASS",
+"FILE_SEGMENT_ELEMENT":"FILE_SEGMENT_ELEMENT",
+"FILE_INFO_BY_HANDLE_CLASS":"FILE_INFO_BY_HANDLE_CLASS",
+"DWORDLONG":"DWORDLONG",
+"EXCEPTION_POINTERS":"EXCEPTION_POINTERS",
+"_EXCEPTION_POINTERS":"_EXCEPTION_POINTERS",
+"WOW64_CONTEXT":"WOW64_CONTEXT",
+"UILANGUAGE_ENUMPROCW":"UILANGUAGE_ENUMPROCW",
+"UILANGUAGE_ENUMPROCA":"UILANGUAGE_ENUMPROCA",
+"TIMEFMT_ENUMPROCW":"TIMEFMT_ENUMPROCW",
+"TIMEFMT_ENUMPROCEX":"TIMEFMT_ENUMPROCEX",
+"TIMEFMT_ENUMPROCA":"TIMEFMT_ENUMPROCA",
+"NUMBERFMTW":"NUMBERFMTW",
+"LOGICAL_PROCESSOR_RELATIONSHIP":"LOGICAL_PROCESSOR_RELATIONSHIP",
+"LOCALE_ENUMPROCW":"LOCALE_ENUMPROCW",
+"LOCALE_ENUMPROCEX":"LOCALE_ENUMPROCEX",
+"LOCALE_ENUMPROCA":"LOCALE_ENUMPROCA",
+"LATENCY_TIME":"LATENCY_TIME",
+"LANGUAGEGROUP_ENUMPROCW":"LANGUAGEGROUP_ENUMPROCW",
+"LANGUAGEGROUP_ENUMPROCA":"LANGUAGEGROUP_ENUMPROCA",
+"GROUP_AFFINITY":"GROUP_AFFINITY",
+"GEO_ENUMPROC":"GEO_ENUMPROC",
+"EXECUTION_STATE":"EXECUTION_STATE",
+"DYNAMIC_TIME_ZONE_INFORMATION":"DYNAMIC_TIME_ZONE_INFORMATION",
+"CURRENCYFMTW":"CURRENCYFMTW",
+"CONTEXT":"CONTEXT",
+"CONSOLE_CURSOR_INFO":"CONSOLE_CURSOR_INFO",
+"CHAR":"char",
+"APPLICATION_RECOVERY_CALLBACK":"APPLICATION_RECOVERY_CALLBACK",
+"HRESULT":"HRESULT",
+"PCNZCH":"char*",
+"LPPROCESS_INFORMATION":"PROCESS_INFORMATION*",
+"LPINT":"int*",
+"LSTATUS":"LSTATUS",
+"DEP_SYSTEM_POLICY_TYPE":"DEP_SYSTEM_POLICY_TYPE",
+"CURRENCYFMTA":"CURRENCYFMTA",
+"va_list":"va_list"}
+
+
+tokens_ignore_prefix = {"APIENTRY", "virtual", "DECLSPEC_NORETURN"}
+
+unknown_types = dict() # we use this dictionary to save and print a list of unknown types
+
+def remove_internal_parentheses(str):
+    while True: # for the all parentheses in the string
+        end = str.find(")") # find the last parenthesis
+        if end == -1: # exit in case of no parenthesis
+            break
+        start = str.rfind("(", 0, end) # looking back for the first parenthesis
+        if start == -1: # exit in case of no parenthesis
+            break
+        new_str = str[:start] # remove everything after (
+        new_str += str[end+1:] # remove everything before )
+        str = new_str
+    return str
+
+def get_type(type_str):
+    return str(types_map.get(type_str, None))
+
+def parse_return_type(line):
+    '''
+    The function searches for a WinAPI return type in the string specified.
+    @in line - a string with a type to find.
+    @out - returning type.
+    '''
+    if ")" in line:
+        line = remove_internal_parentheses(line)
+
+    line = line.split(" ")
+    func_type = None
+    type_token_count = 0
+    for element in line:
+        if element in tokens_ignore_prefix or element == "" or "__" in element\
+           or (("WIN" in element or "NT" in element) and "API" in element):
+            continue
+        # let's return "None" if we have more than 2 tokens that look like candidates for
```

Comment:
style: either capitalize and punctuate the sentence, or do neither for short phrases (the former is preferred in general for new code)